### PR TITLE
Fixes MSRC and upgrades libraries.

### LINKED ISSUE #51: Current Plugin causes issues when enabling SSL on the standard tcp input plugin
When enabling SSL encryption on a Logstash tcp input plugin and the configuration also has kusto output plugin configured, the following is logged and the logstash service fails to start:
java.lang.NoClassDefFoundError: io/netty/internal/tcnative/CertificateCompressionAlgo

This appears to be a compatibility issue with the Netty Handler version brought in when installing the kusto plugin:
```
./logstash-input-tcp-6.3.3-java/vendor/jar-dependencies/io/netty/netty-handler:
total 0
drwxr-xr-x. 2 root root 6 Jul 25 11:13 4.1.93.Final

./logstash-output-kusto-1.0.6-java/lib/io/netty/netty-handler:
total 0
drwxr-xr-x. 2 root root 44 Aug 21 10:34 4.1.68.Final
```

### PR DESCRIPTION
**Pull request by ag-ramachandran**
Fixes CVE's in OSS libraries commons text

Upgrades libraries and possibly fixes #51 that is dependent on Netty

### CODE REVIEW

File: version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.0.0
```
Comment (AsafMah) [inline version:1]: What is the purpose of this file?

File: build.gradle [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+apply plugin: 'maven-publish'
+
+group "org.logstash.outputs"
+version Files.readAllLines(Paths.get("version")).first()
+
+repositories {
+    mavenCentral()
+}
+dependencies {
+    implementation 'com.azure:azure-core-http-netty:1.13.5'
```
Comment (AsafMah) [inline build.gradle:17]: We need to explicitly declare all of this deps?

File: build.gradle [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+task generateGemJarRequiresFile {
+    doLast {
+        File jars_file = file('lib/logstash-output-kusto_jars.rb')
+        jars_file.newWriter().withWriter { w ->
+            w << "# AUTOGENERATED BY THE GRADLE SCRIPT. DO NOT EDIT.\n\n"
+            w << "require \'jar_dependencies\'\n"
+            configurations.runtimeClasspath.allDependencies.each {
+                w << "require_jar(\'${it.group}\', \'${it.name}\', \'${it.version}\')\n"
+            }
+            // w << "require_jar(\'${project.group}\', \'${project.name}\', \'${project.version}\')\n"
```
Comment (AsafMah) [inline build.gradle:97]: this comment - should be removed?

File: build.gradle [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+        }
+    }
+}
+
+task vendor {
+    doLast {
+        String vendorPathPrefix = "lib"
+        configurations.runtimeClasspath.allDependencies.each { dep ->
```
Comment (AsafMah) [inline build.gradle:105]: Add comments to the file explaining what everything does

File: build.gradle [hunk 5]
```diff
@@ -0,0 +1,120 @@
+import java.nio.file.Files
+import java.nio.file.Paths
+
+import static java.nio.file.StandardCopyOption.REPLACE_EXISTING
+
+apply plugin: 'java'
+apply plugin: 'idea'
+apply plugin: 'maven-publish'
+
+group "org.logstash.outputs"
```
Comment (AsafMah) [inline build.gradle:10]: Just to understand, now instead of building the gem via the gemfile, we have a gradle project?

File: build.gradle (hunk 2 above)
Comment (ag-ramachandran) [reply build.gradle:23]: Unfortunately yes! We can either get the dependencies from the effective pom or from the previous ruby file created during  ruby install that was done previously

File: version (hunk 1 above)
Comment (ag-ramachandran) [reply version:1]: Just externalized the version. The gemspec now refers to this file.

File: build.gradle (hunk 4 above)
Comment (ag-ramachandran) [reply build.gradle:116]: Add some comments as well for these steps

File: build.gradle (hunk 5 above)
Comment (ag-ramachandran) [reply build.gradle:12]: That is right! dependencies come in from gradle and the packaging of these dependencies come from gem

**Review by AsafMah — APPROVED** [review]

**Pull request merged**